testing/features/components/targeted_content.feature

```diff
@@ -3,78 +3,29 @@ Feature: Targeted Content component
-      Then a targeted content title is present
-      And the expand/collapse button will indicate it will expand
-
-    Scenario: Welsh Content has a clear title and expand button
-      Given the language is Welsh
-      Then a targeted content title is present
-      And the expand/collapse button will indicate it will expand
-
-    Scenario: English Content can be expanded to reveal more information
-      When I expand the targeted content
-      Then I can see additional information
-      And the expand/collapse button will indicate it will collapse
-      And I can see a close button
-
-    Scenario: Welsh Content can be expanded to reveal more information
-      Given the language is Welsh
-      When I expand the targeted content
-      Then I can see additional information
-      And the expand/collapse button will indicate it will collapse
-      And I can see a close button
-
-    Scenario: Content can be collapsed after being expanded
-      When I expand the targeted content
-      And I collapse the targeted content
-      Then I can no longer see additional information
-
-    Scenario: Content can be closed after being expanded
-      When I expand the targeted content
-      And I close the targeted content
-      Then I can no longer see additional information
-
-  Rule: Anchored Targeted Content
-    Background:
-      Given an Anchored Targeted Content component is on the page
-
-    Scenario: English Content has a clear title and expand button
-      Then a targeted content title is present
-      And the expand/collapse button will indicate it will expand
-
-    Scenario: Welsh Content has a clear title and expand button
-      Given the language is Welsh
-      Then a targeted content title is present
-      And the expand/collapse button will indicate it will expand
-
-    Scenario: English Content can be expanded to reveal more information
-      When I expand the targeted content
-      Then I can see additional information
-      And the expand/collapse button will indicate it will collapse
-      And I can see a close button
-
-    Scenario: Welsh Content can be expanded to reveal more information
-      Given the language is Welsh
-      When I expand the targeted content
-      Then I can see additional information
-      And the expand/collapse button will indicate it will collapse
-      And I can see a close button
-
-    Scenario: Content can be collapsed after being expanded
-      When I expand the targeted content
-      And I collapse the targeted content
-      Then I can no longer see additional information
-
-    Scenario: Content can be closed after being expanded
-      When I expand the targeted content
-      And I close the targeted content
-      Then I can no longer see additional information
-
-    Scenario: Content can be be auto-opened
-      When I jump to the targeted content
-      Then I can see additional information
+  Background:
```

Comment:
@luke-hill Drawing your attention to these changes in particular. Added context in the commit description here https://github.com/citizensadvice/design-system/pull/1141/commits/e081367665511b4919ed8368bc4c0c07a8d27912 but might be worth chatting about it too just so you understand the wider goal here.

Reply:
for parity can we keep the cuke / file name anchored.

Reply:
Not sure I follow. Do you mean the "Given an anchored targeted content component is on the page" wording?

Reply:
yep, basically that.